Sprint-2/3-mandatory-implement/1-bmi.js

```diff
@@ -16,4 +16,19 @@
 
 function calculateBMI(weight, height) {
```

Comment:
Looking good!

What could we rename these parameters to make it super clear what the units are?